<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-A2U04PH" name="com.automation.runners.TestRunner" tests="2" failures="1" timestamp="2022-04-11T17:25:44 EET" time="61.903" errors="0">
  <testcase name="runScenario" time="47.862" classname="com.automation.runners.TestRunner">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.automation.step_definitions.loginStepDefinitions.success_login(loginStepDefinitions.java:37)
at âœ½.user login to the system successfully(file:///C:/Users/HP/Documents/project_two_report/project_two_report/src/main/resources/features/F01_Login.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="14.041" classname="com.automation.runners.TestRunner"/>
  <system-out/>
</testsuite> <!-- com.automation.runners.TestRunner -->
